awscli/customizations/s3/s3handler.py

```diff
@@ -400,6 +400,7 @@ def __init__(self, session, params, result_queue=None,
         # Restrict the maximum chunks to 1 per thread.
         self.config.max_in_memory_upload_chunks = \
             self.MAX_IN_MEMORY_CHUNKS
+        self.config.max_in_memory_upload_chunks
```

Comment:
You are not setting anything here and you need to use ``config.max_in_memory_download_chunks``